SPRIV-Tools doesn't compile with GCC 8.1

[build.log](https://github.com/KhronosGroup/SPIRV-Tools/files/1970660/build.log)

/var/tmp/portage/dev-util/spirv-tools-9999/work/spirv-tools-9999/source/text_handler.cpp: In member function ‘spv_result_t libspirv::AssemblyContext::binaryEncodeString(const char*, spv_instruction_t*)’:
/var/tmp/portage/dev-util/spirv-tools-9999/work/spirv-tools-9999/source/text_handler.cpp:316:10: error: ‘char* strncpy(char*, const char*, size_t)’ output truncated before terminating nul copying as many bytes from a string as its length [-Werror=stringop-truncation]
   strncpy(dest, value, length);
   ~~~~~~~^~~~~~~~~~~~~~~~~~~~~
/var/tmp/portage/dev-util/spirv-tools-9999/work/spirv-tools-9999/source/text_handler.cpp:298:31: note: length computed here
   const size_t length = strlen(value);
                         ~~~~~~^~~~~~~

--- a/source/text_handler.cpp
+++ b/source/text_handler.cpp
@@ -313,7 +313,7 @@ spv_result_t AssemblyContext::binaryEncodeString(const char* value,
   pInst->words.back() = 0;
 
   char* dest = (char*)&pInst->words[oldWordCount];
-  strncpy(dest, value, length);
+  strncpy(dest, value, length + 1);
 
   return SPV_SUCCESS;
 }

